src/Provider/DropboxResourceOwner.php

```diff
@@ -30,7 +30,7 @@ public function __construct(array $response = array())
      */
     public function getId()
     {
-        return $this->response['uid'] ?: null;
+        return $this->response['account_id'] ?: null;
```

Comment:
Can we include the [`ArrayAccessorTrait`](https://github.com/thephpleague/oauth2-client/blob/master/src/Tool/ArrayAccessorTrait.php) in this class and utilize the `getValueByKey ` method to get the values for each of these methods?

```php
use ArrayAccessorTrait;
//
return $this->getValueByKey($this->response, 'account_id')
```